src/tracking.js

```diff
@@ -204,4 +204,10 @@ function wikiaJWPlayerTracking(playerInstance, willAutoplay, tracker) {
 			label: 'autoplay-' + (data.enabled ? 'enabled' : 'disabled')
 		});
 	});
+
+	playerInstance.on('captionsSelected', function (data) {
```

Comment:
Please, add the tracking event to the confluence doc 🌵

Reply:
to be added after merging